hack/generate/samples/internal/go/memcached_with_webhooks.go

```diff
@@ -93,6 +94,9 @@ func (mh *MemcachedGoWithWebhooks) Run() {
 	mh.uncommentKustomizationFile()
 
 	pkg.RunOlmIntegration(mh.ctx)
+
+	// Clean up built binaries, if any.
+	pkg.CheckError("cleaning up", os.RemoveAll(filepath.Join(mh.ctx.Dir, "bin")))
```

Comment:
+1 to removing binaries.

Reply:
Also, we should add this to the gitignore. @camilamacedo86 is doing that in her PR, so no change needed, just FYI.

Reply:
I think we will need anyway. To solve the problem in the sanity I added `git diff --exit-code -- . ':!testdata/go/memcached-operator/bin/*'` since the bin will be gen again in  the CI process. 

However, the sanity sh will be also be removed in the https://github.com/operator-framework/operator-sdk/pull/4023